### ISSUE TITLE
quality selector disabled

### ISSUE THREAD

**Issue by tippfehlr** [jellyfin/jellyfin-vue#2281]
### Description of the bug

The quality selector is disabled.

See #2226 for the original issue.

### Steps to reproduce

See #2226

### Expected behavior

Being able to select the playback quality.

### Platform

Linux

### Browser

Firefox

### Jellyfin server version

10.8.13

**Comment by tippfehlr** [comment]
I believe #2226 was closed inappropriately because the pull request linked to it was merged. I also didn't find a way of reopening it. (Hence this issue)

**Comment by ferferga** [comment]
@tippfehlr Yes, due to merging the pull request. Only admins in the repo are able to reopen it.

Given you already opened this one, we will use this one for tracking this to avoid duplicates.

**Comment by tippfehlr** [comment]
Ah sorry, then I could have just commented.  Now it can be linked to a second pr though :)
